app/views/orders/_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <br>
   <div class="form-group">
       <%= form.label :travelers, class:'form-label mb-2 mt-3' %><br />
-      <%= form.select :travelers,User.all.traveler.pluck(:first_name),{prompt:'Select'},{class: "form-control"}%>
+      <%= form.select :travelers,User.traveler.pluck(:first_name),{prompt:'Select'},{class: "form-control"}%>
   <div class="form-group">
       <%= form.label :from, class:'form-label mb-2 mt-3' %><br />
       <%= form.select :from,Journey.all.pluck(:from),{prompt:'Select'},{class: "form-control"}%>
```

Comment:
same instance variable technique for this line too.

Reply:
okay

Reply:
fixed